packages/react-button/src/components/Button/Button.types.ts

```diff
@@ -110,6 +110,8 @@ export type ButtonProps = ComponentProps &
  */
 export interface ButtonState extends ButtonProps {
   buttonRef?: React.RefObject<HTMLButtonElement>;
+
+  'data-is-focusable': boolean;
```

Comment:
**q:** why was this change needed ?

Reply:
This is set in `useButtonState`:
https://github.com/microsoft/fluentui/blob/983940e05d287b0affc0c2ccbeef89415843ef35/packages/react-button/src/components/Button/useButtonState.ts#L17

---

It's an internal identifier of `FocusZone`: https://github.com/microsoft/fluentui/blob/233dc913eb2348a84dd354aed0617942001d038e/packages/react-focus/src/components/FocusZone/FocusZone.tsx#L33

Reply:
interesting, so is this needed for scope of this PR ?

Reply:
ah nvm it is because now the `any` is gone. ty!